### DISCUSSION TITLE
Partial file download

### DISCUSSION THREAD

**Discussion by TonyTilley** [aclap-dev/video-downloadhelper#479] [Help and Q&A] [answered]
### Your question

I frequently get files only downloaded partially. Is there a way for it to resume partially downloaded files so I don't have to start over every time? The Addon & CoApp version and details identifies my browser as Chrome, but if it's important, I'm specifically on the Brave browser.

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): b470e713751f474c
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\finta",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

**Comment by Season4episode10** [comment, marked as answer]
Nope, unfortunate thing we all experience, if the file is playable, as some partial completed files can be, and the file is say a long file like a movie, I have in the past grabbed the last bit by screen recording it and storing the file in 2 parts, unless both parts are the same resolution and can be joined.

**Reply by TonyTilley** [reply]
ok, thanks. I appreciate your answer.
